Dockerfile

```diff
@@ -20,6 +18,21 @@ RUN dnf upgrade -y \
         nss \
     && dnf clean all
 
+RUN groupadd -g 389 dirsrv \
+    && useradd -u 389 -g 389 \
+        -c 'DS System User' \
+        -d '/var/lib/dirsrv' \
+        -s '/sbin/nologin' dirsrv \
+    && groupadd -g 288 kdcproxy \
+    && useradd -u 288 -g 288 \
+        -c 'IPA KDC Proxy User' \
+        -d '/var/lib/kdcproxy' \
+        -s '/sbin/nologin' kdcproxy
+
+RUN /sbin/ldconfig -X \
+    && : > /etc/fstab \
+    && echo "LANG=C" > /etc/locale.conf
+
```

Comment:
I can't say I'm fond of bundling together commands that really don't have much in common.

When one of them starts to fail, it's harder to see what is going on.